apps/admin-x-activitypub/src/views/Profile/Profile.tsx

```
@@ -373,77 +374,79 @@ const Profile: React.FC<ProfileProps> = ({}) => {
-                            tabs={tabs}
-                            onTabChange={setSelectedTab}
-                        />
+                            <Heading className='mt-4' level={3}>{!isLoadingAccount ? account?.name : <Skeleton className='w-32' />}</Heading>
+                            <span className='mt-1 text-[1.5rem] text-gray-700 dark:text-gray-600'>
+                                <span>{!isLoadingAccount ? account?.handle : <Skeleton className='w-full max-w-56' />}</span>
+                            </span>
+                            {(account?.bio || customFields.length > 0 || isLoadingAccount) && (
+                                <div ref={contentRef} className={`ap-profile-content transition-max-height relative text-[1.5rem] duration-300 ease-in-out [&>p]:mb-3 ${isExpanded ? 'max-h-none pb-7' : 'max-h-[160px] overflow-hidden'} relative`}>
+                                    <div className='ap-profile-content mt-3 text-[1.5rem] [&>p]:mb-3'>
+                                        {!isLoadingAccount ?
+                                            <div dangerouslySetInnerHTML={{__html: account?.bio ?? ''}} /> :
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)